demos/pc/windows/visual_studio/aws_demos.vcxproj

```diff
@@ -9,7 +9,7 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>{C686325E-3261-42F7-AEB1-DDE5280E1CEB}</ProjectGuid>
     <ProjectName>aws_demos</ProjectName>
-    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.15063.0</WindowsTargetPlatformVersion>
```

Comment:
Should we change this?

Reply:
This line should be removed.

Reply:
It we need to change this but not to this value. It is a mistake, I ll change it back

Reply:
fixed